pkg/pool-utils/contracts/BasePool.sol

```diff
@@ -264,7 +274,8 @@ abstract contract BasePool is IBasePool, BasePoolAuthorization, BalancerPoolToke
     function _isOwnerOnlyAction(bytes32 actionId) internal view virtual override returns (bool) {
         return
             (actionId == getActionId(this.setSwapFeePercentage.selector)) ||
-            (actionId == getActionId(this.setAssetManagerPoolConfig.selector));
+            (actionId == getActionId(this.setAssetManagerPoolConfig.selector)) ||
+            super._isOwnerOnlyAction(actionId);
```

Comment:
Note: all these changes were already included in the base of #2059.

The only difference is that at the base of #2059, `BasePool` implements `IControlledPool`. That seems to be just a refactor; the functionality is the same.